ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -59,6 +59,9 @@ struct NotificationDemoView: View {
         VStack {
             Rectangle()
                 .foregroundColor(.clear)
+#if DEBUG
+                .accessibilityIdentifier("Notification View with \(hasTitle ? hasAttribute ? "attributed title \"\(title)\"" : "title \"\(title)\"" : "no title"), \(hasMessage ? hasAttribute ? "attributed message \"\(message)\"" : "message \"\(message)\"" : "no message"), \(image != nil ? "an image" : "no image"), and \(actionButtonTitle != "" ? "an action button titled \"\(actionButtonTitle)\"" : trailingImage != nil ? "a trailing image" : "a dismiss button") in style \(style.rawValue) that is \(isFlexibleWidthToast ? "flexible in width" : "not flexible in width")")
+#endif
```

Comment:
caann we make this easier to read?

Reply:
Personally I also think it's fine if the accessibility identifier isn't an actual sentence, ex. could do "Notification view, hasImage: false, actionButtonTitle: nil, isFlexibleWidthToast: true, etc. if that makes it easier

Reply:
I was thinking more like pretty printing it so that things line up better